rhel8/profiles/ism_o.profile

```diff
@@ -0,0 +1,146 @@
+  - sshd_set_max_auth_tries
+  - sssd_enable_smartcards
+  - accounts_password_minlen_login_defs
+  - accounts_password_pam_minclass
+  - accounts_password_pam_minlen
+  - accounts_password_pam_pwquality
+  - accounts_password_pam_dcredit
+  - accounts_password_pam_lcredit
+  - accounts_password_pam_ocredit
+  - accounts_password_pam_ucredit
+  - accounts_password_pam_maxrepeat
+  - accounts_passwords_pam_faillock_deny
+  - accounts_passwords_pam_faillock_deny_root
+  - accounts_passwords_pam_faillock_interval
+  - accounts_passwords_pam_faillock_unlock_time
+
+  ## Password authentication & Protecting credentials
+  ## Identifiers 1055 / 0418 / 1402
+  - network_nmcli_permissions
+  - configure_kerberos_crypto_policy
+  - kerberos_disable_no_keytab
+  - sebool_kerberos_enabled
+  - sshd_disable_gssapi_auth
+  - enable_ldap_client
+  - set_password_hashing_algorithm_libuserconf
+  - set_password_hashing_algorithm_logindefs
+  - set_password_hashing_algorithm_systemauth
+  - accounts_password_warn_age_login_defs
+  - accounts_maximum_age_login_defs
+  - accounts_minimum_age_login_defs
+ 
+  ## System administration & MFA
+  ## Identifiers 1382 / 1384 / 1386
+  - package_sudo_installed
+  - configure_opensc_card_drivers
+  - force_opensc_card_drivers
+  - sebool_authlogin_yubikey
+
+  ## System patching & Applicatoin versions
+  ## Identifiers 1493 / 1144 / 0940 / 1472 / 1494 / 1495 / 1467 / 1483 
+  - dnf-automatic_apply_updates
+  - package_dnf-plugin-subscription-manager_installed
+  - package_subscription-manager_installed
+
+  ## Centralised logging facility
+  ## Identifiers 1405 / 0988 
+  - rsyslog_cron_logging
+  - rsyslog_files_groupownership
+  - rsyslog_files_ownership
+  - rsyslog_files_permissions
+  - rsyslog_nolisten
+  - rsyslog_remote_loghost
+  - rsyslog_remote_tls
+  - rsyslog_remote_tls_cacert
+  - service_ntpd_enabled
+  - service_ntp_enabled
+  - ntpd_specify_multiple_servers
+  - ntpd_specify_remote_server
+  - service_chronyd_or_ntpd_enabled
+
+  ## Events to be logged
+  ## Identifiers 0584 / 0582 / 0585 / 0586 / 0846 / 0957
+  - display_login_attempts
+  - sebool_auditadm_exec_content
+  - audit_rules_privileged_commands_pam_timestamp_check
+  - audit_rules_privileged_commands
+  - audit_rules_session_events
+  - audit_rules_unsuccessful_file_modification
+  - audit_access_failed
+  - audit_access_success
+
+  ## Web application & Database servers
+  ## Identifiers 1552 / 1277
+  - openssl_use_strong_entropy
+
+  ## Network design and configuration
+  ## Identifiers 1311 
+  - service_snmpd_disabled
+  - snmpd_use_newer_protocol
+
+  ## Wireless networks
+  ## Identifiers 1315 / 1319
+  - wireless_disable_interfaces
+  - network_ipv6_static_address
+
+  ## ASD Approved Cryptopgraphic Algorithims
+  ## Identifiers 1446
+  - enable_dracut_fips_module
+  - enable_fips_mode
+
+  ## Secure Shell access
+  ## Identifiers 1506 / 1449 / 0487 
+  - sshd_version_equal_or_higher_than_74
+  - sshd_allow_only_protocol2
```

Comment:
The rule `sshd_version_equal_or_higher_than_74` doesn't exist, it's a mere OVAL check that is a part of rule `sshd_allow_only_protocol2`. I think that it's enough to keep selected only rule `sshd_allow_only_protocol2`. If you need for some reason to have a separate rule for the SSH version you will have to create rule directory and rule.yml which uses this check.

Reply:
If `sshd_allow_only_protocol2` covers https://www.tenable.com/plugins/nessus/93194, then free to remove.

Removed here https://github.com/wcushen/content/commit/a50c2a6b5f22e8d5dfad450aea07d868ac8c5397

Reply:
Correct me if I am wrong, but I believe that there is no longer a way to configure Protocol version 2 in RHEL8+

Reply:
@redhatrises 

I have seen it cited in a few RHEL 8 profiles:

RH CCP: https://github.com/ComplianceAsCode/content/blob/ebb41ede291d8548b860eb134d88de6a4375a3fd/rhel8/profiles/rht-ccp.profile#L90
CJIS: https://github.com/ComplianceAsCode/content/blob/ebb41ede291d8548b860eb134d88de6a4375a3fd/rhel8/profiles/cjis.profile#L94
HIPAA: https://github.com/ComplianceAsCode/content/blob/4edc9b1192352a2371b60523bdc3dbf900b420cf/rhel8/profiles/hipaa.profile#L52

Reply:
> Correct me if I am wrong, but I believe that there is no longer a way to configure Protocol version 2 in RHEL8+

You are correct.

The rule has a conditional on the SSH version, if OpenSSH is 7.3 or older, it checks for the configuration: https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/ssh/ssh_server/sshd_allow_only_protocol2/oval/shared.xml#L22
For RHEL8+, the rule become just a check for OpenSSH version.

Reply:
@yuumasato Can omit if RHEL 8+ has rebased OpenSSH version 7.8p1 (i.e. no longer supports protocol 1)?

Reply:
@wcushen Yeah, RHEL8+ OpenSSH packages don't provide choice to switch to protocol 1.

If the policy explicitly calls out a check for protocol 2, the rule should probably be there.
Ultimately it is up to you to decide whether to omit the rule or not.

Reply:
@yuumasato If it's cited in other RHEL 8 profile (CJIS, HIPAA, RH CCP) it'd be good to keep to maintain consistency - even it does just represent a check of the version.